src/main/java/com/t4t/thought4thought/utils/Thought4ThoughtResponseObject.java

```diff
@@ -0,0 +1,20 @@
+package com.t4t.thought4thought.utils;
+
+public class Thought4ThoughtResponseObject {
+    private int status; /* 0: success; 1: not successful; -1: unexpected error */
```

Comment:
*Not important*, but ENUMS could be used for the statuses. This way, if we end up adding more later, it'll be easier to keep track of them.

Reply:
Makes sense. I'll add something in my next commit!